operator/deploy/role.yaml

```diff
@@ -8,6 +8,11 @@ metadata:
     app: kiali-operator
     version: ${OPERATOR_VERSION_LABEL}
 rules:
+- apiGroups: ["policy"]
```

Comment:
see the old communications on this part of the yaml: https://github.com/kiali/kiali/pull/1295#discussion_r309305838

I'll repeat it here:

for consistency, use the same syntax and indentation that is used throughout this file. So it should be:

```
- apiGroups: ["policy"]
  resourceNames:
  - kiali-operator
  resources:
  - podsecuritypolicies
  verbs:
  - use
```